.gitignore

```diff
@@ -34,5 +34,4 @@ lerna-debug.log*
 .vscode/*
 !.vscode/settings.json
 !.vscode/tasks.json
-!.vscode/launch.json
```

Comment:
Revert these changes

Reply:
I've added launch file for debugging. Should I commit it?

Reply:
Sure, don't be leaving any IDE related things  lying in the repo.

Reply:
So I should exclude it from `.gitignore`